django/contrib/postgres/forms/jsonb.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(forms.CharField):
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
+    }
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('widget', forms.Textarea)
+        super(JSONField, self).__init__(**kwargs)
+
+    def to_python(self, value):
+        try:
+            return json.loads(value)
+        except ValueError:
```

Comment:
It is handling user supplied data. I'd argue a 500 isn't the worst thing here as it's recieving weird data designed to break the site (or just weird data!). If there was a general security issue here then anyone using `json.loads(request.body)` would have a problem. It may be we want to be more careful, but IANASE (I am not a security expert)